life/updated/02-life.adoc

```diff
@@ -1,189 +1,192 @@
 through the Red Sea, that I might have known how to walk in the right way,
 and to do what the Lord required of me,
-and been in acquaintance and familiarity with my Maker;
-that I might have known when I pleased or displeased the Lord, whom my soul loved,
-but knew not how to become acquainted with him.
-What would I not have parted with for the enjoyment of the Lord,
+and been in acquaintance and familiarity with my Maker!
+Then I might have known when I pleased or displeased the Lord, whom my soul loved,
+but now I know not how to become acquainted with Him.`"
+What would I not have then parted with for the enjoyment of the Lord,
 and assurance of salvation? Surely if it were
 possible for me to have enjoyed all the world,
 I could freely have parted with it,
-for peace and satisfaction to my poor distressed soul, that mourned as without hope.
+for peace and satisfaction to my poor distressed soul that mourned as without hope.
 Many a time, and many hours have I spent alone, reading and mourning, when no eye saw me,
-nor ear heard me, neither could I find comfort in reading,
-because it was a book sealed unto me.
+nor ear heard me. Neither could I find comfort in reading the Scripture,
+for it was a book sealed unto me.
 Then did I mourn and say,
-oh! that I had been born in the days when our
+"`Oh, that I had been born in the days when our
 blessed Savior Jesus Christ was upon the earth!
-How would I have followed him, and sat at his feet, as Mary did;
+How would I have followed Him, and sat at His feet, as Mary did;
 how freely could I have left my father`'s house, and all my relations,
-for true peace and assurance of life eternal for my immortal soul.
+for true peace and assurance of life eternal for my immortal soul.`"
 
 Under this exercise I grew very sad,
 insomuch that my mother apprehended I was going into a consumption,
-and greatly feared my death; and would say to me,
-"`Can you take delight in nothing? I would have you
+and greatly fearing my death, would say to me,
+"`Can you take delight in nothing? I would rather you
 walk forth into the fields with the young people,
 for recreation, and delight yourself in something.`"
-And to please her, I have sometimes, when we were out of our employment,
+And to please her, I have sometimes, when we were finished with our employment,
 gone forth with sober young people, but I found no comfort in that.
-Then I fell into a custom of reading the Scriptures, alone in private,
+Then I fell into a custom of reading the Scriptures in private,
 reading and crying,
-because I knew not that heavenly power and spirit to have dominion in me,
-that was in them who gave forth the Scriptures;
-and nothing else but the substance would give me true satisfaction:
-therefore the Scripture was but a book sealed unto me.
+because I knew not that heavenly power and Spirit to have dominion in me
+that was in them who gave forth the Scripture,
```

Comment:
I would keep this plural - Scriptures.  They usually speak of the ScriptureS and not just the Scripture. And also, its plural right below in line 87. 👍🏻